api-server/config/policies.js

```diff
@@ -5,24 +5,29 @@ module.exports.policies = {
   'UserController': {
     'create': true, // We dont need authorization here, allowing public access
     'find': ['isAuthenticated', 'isAdmin', 'addDeletedParam'],
```

Comment:
@grossmail1 adding this middleware `addDeletedParam` as a policy so all requests to the UserController (/user endpoint) get the param added on to the request `deleted = false` so that the results sent back are only our "sudo" active records. This param isn't applied/added if the request explicitly sets `deleted = true|false` ie: if you're trying to view deleted records intentionally.